doc/build_docs.rst

```diff
@@ -10,6 +10,7 @@ The FlamingPy documentation is built using `sphinx`. To build the documentation
 * `numba <https://numba.pydata.org/>`_ >= 0.53.1
 * `NumPy <http://numpy.org/>`_ >= 1.21.0
 * `pandas <https://pandas.pydata.org/>`_ >= 1.2.1
+* `pylint <https://pypi.org/project/pylint/>`_ >= 2.13.9
```

Comment:
Do we really need `pylint` to build the docs?

Reply:
I added `pylint` here because the `conf.py` file uses `pyreverse` ([here](https://github.com/BestQuark/flamingpy/blob/3263d0fccb68156d45e1c4ddccc633a1112bc2fa/doc/conf.py#L159)) to automatically generate the uml diagrams that are displayed in the fp page.

Reply:
Ah, yes, makes sense. I'll mark resolved.